grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -688,7 +688,7 @@ class ReportController {
                 }
                 def replenishmentPeriodDays = params.replenishmentPeriodDays ?: inventoryLevel && inventoryLevel.replenishmentPeriodDays ? inventoryLevel.replenishmentPeriodDays : 365
                 def leadTimeDays = params.leadTimeDays ?: inventoryLevel && inventoryLevel.expectedLeadTimeDays ? inventoryLevel.expectedLeadTimeDays : 365
-                def productExpiry = forecastingService.getProductExpiry(Location.load(params.originId), replenishmentPeriodDays + leadTimeDays, item.productId)
+                def productExpiry = forecastingService.getProductExpiry(Location.load(params.originId), replenishmentPeriodDays.toInteger() + leadTimeDays.toInteger(), item.productId)
```

Comment:
@MagdaToczek Could you explain the need for this?

Reply:
Sure, in previous version it was concatenation of strings... And that wasn't my intention :)

Reply:
I corrected this a bit and change to direct typing of this two variables.

Reply:
Ah ok that makes sense.